abletonosc/device.py

```diff
@@ -85,6 +88,42 @@ def device_get_parameter_value(device, params: Tuple[Any] = ()):
             # https://github.com/ideoforms/AbletonOSC/issues/33
             param_index = int(params[0])
             return param_index, device.parameters[param_index].value
+        
+        # Uses str_for_value method to return the UI-friendly version of a parameter value (ex: "2500Hz")
```

Comment:
This might be a standard for Live devices, but VSTs and AUs wouldn't necessarily follow this standard even though their readable strings are also available through this method. I've updated the examples to have a space, but further suggestions for usage might cause confusion.